frontends/concrete-python/tests/compilation/test_modules.py

```diff
@@ -602,3 +602,74 @@ def dec(x):
     # Make sure computation happened in simulation.
     assert isinstance(module.dec.runtime, SimulationRt)
     assert isinstance(encrypted_result, int)
+
+
+def test_trace_wire_single_input_output(helpers):
+    @fhe.module()
+    class Module:
+        @fhe.function({"x": "encrypted"})
+        def a(x):
+            return (x * 2) % 20
+
+        @fhe.function({"x": "encrypted"})
+        def b(x):
+            return (x * 2) % 50
+
+        @fhe.function({"x": "encrypted"})
+        def c(x):
+            return (x * 2) % 100
+
+        composition = fhe.Wired([])
```

Comment:
Will this be optional?

Reply:
The default composition policy is `FullyWired`, which puts everything into the same partition. I have an assert to check that the user picked the proper composition policy before using wire tracing. 

An other option is to change the composition policy on its behalf when wire tracing gets activated.

WDYT ?